### DISCUSSION TITLE
Sort imports in all files from command line

### DISCUSSION THREAD

**Discussion by medihack** [astral-sh/ruff#3514] [General]
I use ruff together with the VS Code extension to also sort my imports on saving a file. Is it also possible to sort all imports from all files on the command line? `ruff . --fix` doesn't seem to sort the imports.

**Comment by charliermarsh** [comment]
Can you try running `ruff . --fix --select I`? You'll need to enable the isort rules (`I` prefix) in order to get import sorting to run from the CLI.

**Comment by medihack** [comment]
Thanks a lot, that solves it. And it's very well documented in https://beta.ruff.rs/docs/rules/ My fault :-(
